.devcontainer/Dockerfile.base

```diff
@@ -0,0 +1,42 @@
+FROM runpod/pytorch:2.4.0-py3.11-cuda12.4.1-devel-ubuntu22.04
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    wget \
+    nano \
+    socat \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Miniconda (optional, if you want it ready at build-time)
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda.sh \
+   && bash /tmp/miniconda.sh -b -p /miniconda3 \
+   && rm /tmp/miniconda.sh
+
+ENV PATH="/miniconda3/bin:${PATH}"
+RUN eval "$(/miniconda3/bin/conda shell.bash hook)"
+RUN conda create -n comfyui python=3.11 -y
+RUN conda create -n comfystream python=3.11 -y
+COPY . /app
```

Comment:
Actually another comment is if you think it would be ok to change this `/app` path to something else. The `/app` will conflict with the workdir of the `ai-worker` image itself (not only for comfyui).

I think we could copy the `comfystream` repo and set it up on the `/comfystream` path, corresponding to the `/ComfyUI`. WDYT?

Reply:
`/workspace` in not available in the base container, it is only created by the [devcontainer configuration](https://github.com/yondonfu/comfystream/pull/20/files#diff-24ad71c8613ddcf6fd23818cb3bb477a1fb6d83af4550b0bad43099813088686R41), so we should leave `/workspace` open for the current working directory to mount. 

I can use `/comfystream` instead of `/app` for the temp files that are copied for the build. Current the devcontainer doesn't need anything from `/app` in the base